src/components/UserList.jsx

```diff
@@ -1,15 +1,24 @@
+import { useNavigate } from "react-router";
 import UserCard from "./UserCard";
 
 function UserList({ users }) {
+  const navigate = useNavigate();
+
+  const handleCardClick = (id) => {
+    navigate(`/post/${id}`);
+  };
+
   return (
     <ul className="tablet:grid-cols-[repeat(auto-fill,_minmax(186px,1fr))] pc:grid-cols-4 grid w-full max-w-928 grid-cols-2 justify-center gap-16">
       {users.map((user) => {
-        const { imageSource, name, questionCount } = user;
+        const { id, imageSource, name, questionCount } = user;
 
         return (
           // 카드 너비 반응형 구현 필요
           <li key={user.id}>
```

Comment:
id를 구조 분해 했다면 이부분도 그냥 id로 바꾸는 게 맞지 않나 싶네요?
```suggestion
        const { id, imageSource, name, questionCount } = user;

        return (
          // 카드 너비 반응형 구현 필요
          <li key={id}>
```

Reply:
수정했습니다!